docker-compose.yml

```diff
@@ -237,7 +237,7 @@ services:
       "
 
   indexer:
-    image: joystream/hydra-indexer:v5.0.0-alpha.1
+    image: hydra-indexer
```

Comment:
I think like me you were using local build of hydra-indexer ;)

Reply:
Yeah ;), fixed that in https://github.com/Joystream/joystream/pull/5001/commits/190daa0cb75ed5ec88e8c7b7a470daa125cab15b